docs/TextColumns.md

```diff
@@ -7,12 +7,25 @@ _New in [:octicons-tag-24: 2.7.6](https://github.com/py-pdf/fpdf2/blob/master/CH
 
 The `FPDF.text_columns()` method allows to create columnar layouts, with one or several columns. Columns will always be of equal width.
 
+### Parameters ###
+
 Beyond the parameters common to all text regions, the following are available for text columns:
 
 * l_margin (float, optional) - override the current left page margin.
 * r_margin (float, optional) - override the current right page margin.
 * ncols (float, optional) - the number of columns to generate (Default: 2).
 * gutter (float, optional) - the horizontal space required between each two columns (Default 10).
+* balance (bool, optional) - Create height balanced columns, starting at the current height and ending at approximately the same level.
+
+### Methods ###
+
+Text columns support all the standard text region methods like `.paragraph()`, `.write()`, `.ln()`, and `.render()`. In addition to that:
+
+* `.new_column()` - End the current column and continue at the top of the next one.
+
+A FORM_FEED character (`\u000c`) in the text will have the same effect as an explicit call to `.new_column()`,
```

Comment:
What's the purpose of this behaviour?

Reply:
No, this is the originally intended purpose of this character since the time of noisy "line printers". A lot of software accepts it for triggering page breaks, and you'll find it in many documents for that reason. We could actually consider supporting it in other methods as well.

Reply:
Alright